docs/ardana/testing.md

```diff
@@ -22,7 +22,7 @@ limits in our project.
 
 To be able to use the Jenkins job and login to a deployed env, you need:
 
-* the login creds for https://ci.nue.suse.com
+* the cloud team login creds for https://ci.nue.suse.com (ask in the cloud-team RocketChat channel if you don't know what it is, and someone will PM you)
```

Comment:
I suggest making the channel a hyperlink like this: [the `cloud-team` Rocket.Chat channel](https://chat.suse.de/channel/cloud-team).